### PR TITLE
Onboarding: re-engagement

### PR DESCRIPTION
**Pull request by Buhle79**

### CODE REVIEW

File: Onboarding/re-engagement.md [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+
+```stack
+trigger(on: "MESSAGE RECEIVED") when has_only_phrase(event.message.text.body, "engage")
+
+```
+
+```stack
+# card Setup, then: FetchError do
+#   update_contact(reengagement_message: "1st message")
+# end
```
Comment (HawkiesZA) [inline Onboarding/re-engagement.md:11]: Do we need this if it's all commented out?

File: Onboarding/re-engagement.md [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+  button_error_text = page.body.body.text.value.message
+end
+
+```
+
+## Drop Off Redirect And Schedule Reminder
+
+```stack
+card DropOffRedirect when contact.reengagement_message == "1st message", then: DropOff2ndReminder do
+  text("To send 2nd message")
```
Comment (HawkiesZA) [inline Onboarding/re-engagement.md:50]: Probably need to remove this?

File: Onboarding/re-engagement.md [hunk 3]
```diff
@@ -0,0 +1,431 @@
...
+  log("To send 2nd message")
+  schedule_stack("b11c7c9c-7f02-42c1-9f54-785f7ac5ef0d", in: 23 * 60 * 60)
+end
+
+card DropOffRedirect when contact.reengagement_message == "2nd message", then: DropOff3rdReminder do
+  text("To send 3rd message")
```
Comment (HawkiesZA) [inline Onboarding/re-engagement.md:56]: Probably need to remove this?

File: Onboarding/re-engagement.md [hunk 4]
```diff
@@ -0,0 +1,431 @@
...
+  log("To send 3rd message")
+end
+
+card DropOffRedirect when contact.reengagement_message == "remind me", then: ReminderRequest do
+  text("Remind me tomorrow message")
```
Comment (HawkiesZA) [inline Onboarding/re-engagement.md:61]: Probably need to remove this?

File: Onboarding/re-engagement.md [hunk 5]
```diff
@@ -0,0 +1,431 @@
...
+  log("Remind me tomorrow message")
+end
+
+card DropOffRedirect, then: DropOff1stReminder do
+  text("To send 1st message")
```
Comment (HawkiesZA) [inline Onboarding/re-engagement.md:66]: Probably need to remove this?

File: Onboarding/re-engagement.md [hunk 6]
```diff
@@ -0,0 +1,431 @@
...
+```
+
+## Schedule for 1 after later
+
+Need to set this at the start od onboarding
+
+```stack
+card Schedule1hLater do
+  schedule_stack("b11c7c9c-7f02-42c1-9f54-785f7ac5ef0d", in: 60 * 60)
+end
```
Comment (HawkiesZA) [inline Onboarding/re-engagement.md:429]: I think this needs to move into Intro & Onboarding right at the top (either just before or just after FetchError).

**Review by HawkiesZA — CHANGES_REQUESTED** [review]
Just a few small changes.

File: Onboarding/re-engagement.md (hunk 2 above)
Comment (Buhle79) [reply Onboarding/re-engagement.md:50]: Hi @HawkiesZA why do we need to remove these checks as they are for redirecting which message we need to send to a user?

**Review by HawkiesZA — APPROVED** [review]

**Pull request merged**